### PR TITLE
chore: update messaging for Deploy tab

### PR DESCRIPTION
**Pull request by IAmAnubhavSaini**
## Description

Update messaging for deploy tab.

Fixes # (issue)

> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by IAmAnubhavSaini** [comment]
Screenshot 2022-07-01 at 10 06 36 AM

**Comment by IAmAnubhavSaini** [comment]
Tests pass.

Screenshot 2022-07-01 at 2 50 21 PM

File: app/client/src/pages/Editor/gitSync/components/GitChangesList.test.tsx [hunk 1]
```diff
@@ -21,64 +21,8 @@ describe("GitChangesList", () => {
...
-          },
-          _owner: null,
-          _store: {},
-        },
-      ];
-      const expectedJSON = JSON.stringify(expected);
+      const expectedJSON =
+        '[{"key":"change-status-widget","ref":null,"props":{"message":"1 page modified","iconName":"widget","hasValue":true},"_owner":null,"_store":{}},{"key":"change-status-query","ref":null,"props":{"message":"1 query modified","iconName":"query","hasValue":true},"_owner":null,"_store":{}},{"key":"change-status-js","ref":null,"props":{"message":"1 JS Object modified","iconName":"js","hasValue":true},"_owner":null,"_store":{}},{"key":"change-status-database-2-line","ref":null,"props":{"message":"1 datasource modified","iconName":"database-2-line","hasValue":true},"_owner":null,"_store":{}},{"key":"change-status-git-commit","ref":null,"props":{"message":"1 commit ahead. These are the commits that haven\'t been pushed to remote yet.","iconName":"git-commit","hasValue":true},"_owner":null,"_store":{}},{"key":"change-status-git-commit","ref":null,"props":{"message":"1 commit behind. We will try to pull before pushing your changes.","iconName":"git-commit","hasValue":true},"_owner":null,"_store":{}}]';
```
Comment (ankitakinger) [inline app/client/src/pages/Editor/gitSync/components/GitChangesList.test.tsx:25]: why did we change this? It won't be easy to read for developers working on this right?

File: app/client/src/pages/Editor/gitSync/components/GitChangesList.tsx [hunk 2]
```diff
@@ -1,15 +1,21 @@
 import React from "react";
 import styled from "constants/DefaultTheme";
 import { Classes } from "components/ads/common";
+import Icon, { IconSize } from "components/ads/Icon";
 import Text, { TextType } from "components/ads/Text";
 import { Colors } from "constants/Colors";
-import Icon, { IconSize } from "components/ads/Icon";
 import { useSelector } from "react-redux";
```
Comment (ankitakinger) [inline app/client/src/pages/Editor/gitSync/components/GitChangesList.tsx:7]: The ads imports can be merged into one import.

Comment (IAmAnubhavSaini) [reply app/client/src/pages/Editor/gitSync/components/GitChangesList.tsx:7]: No. This breaks for some reason.

File: app/client/src/pages/Editor/gitSync/components/GitChangesList.test.tsx (hunk 1 above)
Comment (IAmAnubhavSaini) [reply app/client/src/pages/Editor/gitSync/components/GitChangesList.test.tsx:25]: Earlier I created that object from this json itself. Now I am just removing the middle man.

**Review by ankitakinger — DISMISSED** [review]

**Pull request merged**